pkg/util/http.go

```diff
@@ -0,0 +1,16 @@
+package util
+
+import "regexp"
+
+const (
+	apiKeyReplacement = "api_key=*************************$1"
+)
+
+var apiKeyRegExp = regexp.MustCompile("api_key=*\\w+(\\w{5})")
```

Comment:
What is the purpose of the `*` after `=` ? Do we want to match `api_key=====fakeapikey` ?

Reply:
I don't think there's a good reason. We might want to revisit this. I'll merge as-is: we have a card in our backlog to filter apikeys directly at the logger level, I'll add a comment there.